cost/low_account_usage/low_account_usage.pt

```diff
@@ -226,12 +256,12 @@ policy "policy_budget_alert" do
 | Billing Center Name | Vendor | Account Name | Savings |
```

Comment:
I think there would be some value in providing the defined parameters here for billing centers and thresholds.